libsolutil/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ set(sources
 )
 
 add_library(solutil ${sources})
-target_link_libraries(solutil PUBLIC jsoncpp Boost::boost Boost::filesystem Boost::system)
+target_link_libraries(solutil PUBLIC jsoncpp Boost::boost Boost::filesystem Boost::system range-v3)
```

Comment:
It's not header-only?

Reply:
It is - this pulls in the include directory only - it's an "INTERFACE" library for cmake - that's what they call header-only libraries.

Reply:
Ah right!

Reply:
i would like get merged

-